cuda/test/matrix/dense_kernels.cpp

```diff
@@ -358,6 +380,19 @@ TEST_F(Dense, AdvancedApplyIsEquivalentToRef)
 }
 
 
+TEST_F(Dense, AdvancedApplyMixedIsEquivalentToRef)
+{
+    set_up_apply_data();
+
+    x->apply(convert<MixedMtx>(alpha).get(), convert<MixedMtx>(y).get(),
+             convert<MixedMtx>(beta).get(), convert<MixedMtx>(expected).get());
+    dx->apply(convert<MixedMtx>(dalpha).get(), convert<MixedMtx>(dy).get(),
+              convert<MixedMtx>(dbeta).get(), convert<MixedMtx>(dresult).get());
+
+    GKO_ASSERT_MTX_NEAR(dresult, expected, 1e-14);
```

Comment:
Should this be 1e-7, or can the previous test also be checked to 1e-14?
```suggestion
    GKO_ASSERT_MTX_NEAR(dresult, expected, 1e-7);
```